### DISCUSSION TITLE
fail to show trace  in browser tracing, instead on show page metrics, like loading latency.

### DISCUSSION THREAD

**Discussion by szhengli** [apache/skywalking#9793] [General]
not show browser trace. my project is a VUE SPA,   not to  mention xhr invoking. here are some configs for reference, pls advise, thanks

1.vue
ClientMonitor.register({

  collector: 'http://127.0.0.1:8010', 
  service: 'login-ui',
  pagePath: location.href,
  vue: true,
  enableSPA: true,
  traceSDKInternal: true,
  serviceVersion: 'v1.0.0',
});

2、skywalking snapshot, where fail to  show the trace info of the  frontend url endpoint.

![image](https://user-images.githubusercontent.com/16127498/196086801-41098470-2dd4-4026-9520-03def66ddd1a.png)

**Comment by baixjian** [comment]
Have you solved the problem？
